Pipfile

```diff
@@ -18,6 +18,8 @@ pandas = "*"
 parse = "*"
 prettyprinter = "*"
 thoth-lab = "*"
+thoth-analyzer = "*"
+thoth-storages = "*"
```

Comment:
Add the new dependencies also to `requirements.txt`

Reply:
Sure, thanks for the reminder.

Reply:
What is the above failure related to @CermakM ?

Reply:
@harshad16 looks like it's trying to load JSON content but it's just a raw text file (see the [fixtures](/fixtures))

Reply:
@CermakM  yes build log is raw text, but the input which would come from user-api would have metadata with it, that's the reason its a JSON. 

Anyway, how did you arrive at that error? Can you comment the step followed, as re text each of the sub-commands from CLI and none gave me this error!

Reply:
@harshad16 I issued this command: `thoth-build-analysis report fixtures/thoth-lab.pip.failed.log`

If we expecta a JSON on the input, then we should account for that in the CLI and add a preprocessing step. ASFAIR, the original purpose of the api was to analyse *raw* build logs, which is what `thoth-build-analysis report --help` also states.

Reply:
Thanks for the comment, the purpose of the API was to analyze raw build logs is correct but we wanted to have some metadata with it when storing in the storage. 
I have modified the code to accommodate the behavior, it will take raw build logs. 
I will update the pr :smile: